opencog/spacetime/octomap/OctoMapValue.cc

```diff
@@ -0,0 +1,69 @@
+#include "OctoMapValue.h"
+
+#include <opencog/spacetime/octomap/atom_types.h>
+#include <opencog/util/exceptions.h>
+#include <opencog/atoms/base/ClassServer.h>
+#include <opencog/atoms/value/ValueFactory.h>
+
+using namespace opencog;
+
+
+void OctoValue::update()
+{
+    //point3d p = (_om->get_locations_of_atom(_octo_atom)).front();
+    //_value = {p.x(), p.y(), p.z()};
+    _om->insert_atom(point3d(_value[0], _value[1], _value[2]), _item);
+}
+
+OctoValue::OctoValue(const HandleSeq& hseq,
+                     std::vector<double> values) : FloatValue(OCTO_VALUE, values)
+{
+
+    if( hseq.size() != 2) {
+        throw InvalidParamException(TRACE_INFO, 
+                "Required 2 arguments. Provided %d.", hseq.size());
+    }
+
+    _item = hseq[0];
+    _octo_atom = hseq[1];
```

Comment:
Could you rename `_octo_atom` to `_octomap_node`?  This would be a little less confusing, because right now, everything is octo-this and octo-that, and its hard to keep in mind what is what.